tests/math.test.ts

```diff
@@ -149,7 +149,7 @@ $$</p>`;
   expect(received).toBe(expected);
 });
 
-it('HTML header and body', () => {
+it('HTML heading and body', () => {
```

Comment:
```suggestion
it('HTML head and body', () => {
```

ここは見出しではなくてHTMLの `<head>` 要素のことなので、 "heading" ではなくて "head" が正しいでしょう。

Reply:
それ以外は大丈夫です